src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -316,13 +316,13 @@ public void periodic() {
       logDesiredPosition.append(desiredState.position);
       logDesiredVelocity.append(desiredState.velocity);
       logCurrentVoltage.append(currentVoltage);
-    } else {
-      if (goalState.position == MIN_HEIGHT
-          && !MathUtil.isNear(MIN_HEIGHT, currentState.position, STOWED_POSITION_TOLERANCE)
-          && currentState.velocity == 0) {
-        encoder.reset();
-      }
-    }
+    } /*else {
+        if (goalState.position == MIN_HEIGHT
+            && !MathUtil.isNear(MIN_HEIGHT, currentState.position, STOWED_POSITION_TOLERANCE)
+            && currentState.velocity == 0) {
+          encoder.reset();
+        }
+      } */
```

Comment:
Remove before merging
```suggestion
    }
```